packages/eslint-plugin-expo/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "eslint-plugin-expo",
+  "version": "0.0.1",
+  "description": "Eslint rules for Expo apps",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Expo",
+  "main": "./build/index.js",
+  "files": [
+    "README.md",
+    "build"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/expo/expo.git",
+    "directory": "packages/eslint-plugin-expo"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest",
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@typescript-eslint/types": "^7.2.0",
+    "@typescript-eslint/utils": "^7.2.0",
+    "eslint": "^8.57.0"
+  },
+  "devDependencies": {
+    "@types/eslint": "^8.56.5",
+    "@types/jest": "^29.5.12",
+    "@typescript-eslint/rule-tester": "^7.2.0",
+    "eslint": "^8.57.0",
+    "eslint-config-universe": "^12.0.0",
+    "eslint-plugin-eslint-plugin": "^5.0.0",
+    "eslint-plugin-node": "^11.1.0",
```

Comment:
```suggestion
```

This dependency does not need to be specified manually in dev dependencies, it's a regular dependency in "eslint-config-universe" already: * https://github.com/expo/expo/blob/main/packages/eslint-config-universe/package.json#L47